feat(chore): implement revoke token and revoke all token (#222)
fastify/fastify-oauth2: README.md

});
```

> - `revokeToken(Token, tokenType, params, callback)`: A function that use for remove the current access_token or refresh_token. If the callback is not passed it will return a promise. The callback call or promise resolution returns an `void`

Reviewer: ```suggestion
- `revokeToken(Token, tokenType, params, callback)`: A function to revoke the current access_token or refresh_token on server side. If the callback is not passed it will return a promise. The callback call or promise resolution returns `void`
```
